README.md

```diff
@@ -42,6 +40,12 @@ Then launch JupyterLab using
 jupyter lab --watch
 ```
 
-This will automatically recompile `@jupyterlab/toc` upon changes,
-and JupyterLab will rebuild itself. You should then be able to refresh the
-page and see your changes.
+This will automatically recompile `@jupyterlab/toc` upon changes, and JupyterLab will rebuild itself. You should then be able to refresh the page and see your changes.
+
+<!-- links -->
+
+[badge-binder]: https://mybinder.org/badge_logo.svg
+[binder]: https://mybinder.org/v2/gh/kgryte/jupyterlab-toc/demo-notebook?urlpath=lab%2Ftree%2Fnotebooks%2Fdemo.ipynb
```

Comment:
This link should be updated to point to this repo.

Reply:
Ah, I see you already mentioned this above :)

Reply:
Done.